met-api/migrations/versions/cad222167ce7_memberships_added.py

```diff
@@ -0,0 +1,58 @@
+"""memberships added
+
+Revision ID: cad222167ce7
+Revises: 5880bead8f03
+Create Date: 2023-01-24 14:31:48.024605
+
+"""
+from datetime import datetime
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'cad222167ce7'
+down_revision = '5880bead8f03'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    membership_status_codes = op.create_table('membership_status_codes',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('status_name', sa.String(length=50), nullable=True),
+    sa.Column('description', sa.String(length=50), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('membership',
```

Comment:
Should we be specific in naming? Like EngagementMembership rather than membership?

Reply:
good point..i was thinking to extend this for other entities as well in the future ;
like for surveys; if there comes a requirement survey can be edited a few folks etc etc... 
so we can introduce a entity_type here  and manage..
so kept it as just Membership so that we can extend it..

Reply:
absolutely, I thought this is a specific table. If it's already intended to be general then fine as is